resources/js/photoinit.js

```diff
@@ -132,6 +133,8 @@ function initPhotoSwipeFromDOM (gallerySelector) {
         gallery.listen('close', resetMailForm);
 
         gallery.init();
+
+        img = gallery.currItem.src.split('/').pop();

```

Comment:
The further use of `img` is now depending on `parseThumbnailElements` which is a bad idea in my opinion.